test/instrumentations/amqplib/amqplib.test.ts

```diff
@@ -30,6 +30,32 @@ const startRabbitMqContainer = async () => {
     .start();
 };
 
+let warmupState = {
```

Comment:
warmups were happening for each version which made no sense, i've fixed this everywhere relevant